### ISSUE TITLE
Laravel 5.5, blade template in custom component doesn't work properly  

### ISSUE THREAD

**Issue by mrofi** [LaravelCollective/html#502]
Hi, I have an issue with form custom component.
in my blade template, I use blade directive @push to inject a javascript code.
In Laravel 5.3 or 5.4 it works fine, but when I implement in Laravel 5.5 it doesn't . 
I don't the bug came from laravel or this package.

Thank's for any advice.

**Comment by tshafer** [comment]
Please provide example code.

**Comment by tshafer** [comment]
I don't think this has anything to do with the package. Does it work if you remove the package?

**Comment by mrofi** [comment]
I use this package feature : Custom Component : 
https://laravelcollective.com/docs/master/html#custom-components

to create custom form input view.
But in my case, I added directive @ push in my blade template to generate form.

I see no update in your code, 
https://github.com/LaravelCollective/html/blob/5.6/src/Componentable.php#L52

so I think this bug came from laravel when rendering view from blade template in custom component only.
